ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml

```diff
@@ -74,8 +74,9 @@ tests:
     make unit-tests
   container:
     from: src
+  skip_if_only_changed: ^doc/|samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
 - as: test-e2e
-  skip_if_only_changed: ^doc/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
+  skip_if_only_changed: ^doc/|samples/|\.md$|\.properties$|\.copyrightignore$|\.dockerignore$|^(?:.*/)?(?:\.gitignore|\.md$|OWNERS|PROJECT|LICENSE|DCO|manager/OWNERS|agent/OWNERS|operator/OWNERS|release-notes/*)$
```

Comment:
the e2e test depends the kafka and postgres samples. if you remove it, we cannot verify the changes in samples.

Reply:
@clyang82 According to my check, I didn't find any files in e2e that reference samples inside.
Instead, there are some files in `operator/config/samples` that are used by the e2e test.

Reply:
Only skip the `^sample`